lib/plug/basic_auth.ex

```diff
@@ -33,13 +33,15 @@ defmodule Plug.BasicAuth do
   If you want to provide your own authentication logic on top of
   Basic HTTP auth, you can use the low-level functions. For example:
 
-      import Plug.BasicAuth
+      plug :auth
 
```

Comment:
We need to say then they can define their own plug, giving a bit more of context to what we are doing